spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/jdbc/DataSourceProperties.java

```diff
@@ -288,6 +294,8 @@ public String determineUrl() {
 		if (StringUtils.hasText(this.url)) {
 			return this.url;
 		}
+
+		logger.debug("No url defined, looking for embedded database.");
```

Comment:
I don't think adding a logging statement here is part of what is discussed in the issue you've referenced.

Reply:
My idea here was to give more insight into what's happening in the case described in the issue. As far as I understood it is hard to find out why there is this error because of the "magic" happening with with embedded database (i.e. no url means: try to connect to an embedded database) 

But the original issue was about something different anyway, I was trying to do the 
> quote Let's fix the error message though

So I can remove the logging statement if you want.